util/util.go

```diff
@@ -38,6 +38,10 @@ func commonFlags() []cli.Flag {
 			Name:  "table",
 			Usage: "Return output in tabular format. This is the default output format.",
 		},
+		cli.BoolFlag{
+			Name:  "csv",
+			Usage: "Return output in csv format.",
+		},
```

Comment:
I'm indifferent on this. I made each format a flag so that there'd be a discrete set of options and users would be able to bash-complete them. This is yet another design decision where I think either would be fine, but I'd like to open it up to discussion.